### ISSUE TITLE
runtime error 404 - "cloud provider was terminated unexpectedly" with DLL but "no such element" with AddIn

### ISSUE THREAD

**Issue by dornech** [GCuser99/SeleniumVBA#121]
Occasionally when executing element access methods via a webdriver object, I get this error message:
![image](https://github.com/user-attachments/assets/98e6727f-aaa0-41bc-a709-fbc5d815117b)

I have not found out the specific situaitons but I can overcome the problem with sufficient wait time included.

To me it looks like there is some internal wait missing?

**Comment by dornech** [comment]
Ha, tried something: switched to the xls VBA Add in. Same place - same error code - different error message:
![image](https://github.com/user-attachments/assets/bc3316db-cac2-4a77-8678-3b3d9c265318)

This message seems much more logical also keeping in mind that I can overcoem the error with a wait.

The issue occurs after a click to an WebElement that shows a popup. Seems there is a wait for the internal click processing missing.

**Comment by GCuser99** [comment]
Yeah twinBASIC is still in beta and has historically had some issues with error reporting - I will check the details out and get back to you. 

On the actual error - are you using a non-zero ImplicitMaxWait?

**Comment by dornech** [comment]
For impilicitmaxwait I use standard setting from the ini-file - no own value set

**Comment by GCuser99** [comment]
It defaults to 0 (Selenium WebDriver default). Try something longer like `driver.ImplicitMaxWait = 1000`.

**Comment by dornech** [comment]
Ah, I guess that is an possibility I did not consider sufficiently in the past. Thanks!

**Comment by dornech** [comment]
> Yeah twinBASIC is still in beta and has historically had some issues with error reporting - I will check the details out and get back to you.

Thanks again. This might be the reason and I think there are really issues. :-)
I think the example above is not the only one.  Several error messages with the DLL are ... quite surprising meaning wrong.

**Comment by GCuser99** [comment]
I made a tB [issue report](https://github.com/twinbasic/twinbasic/issues/1927) for this - we will just have to be patient...

**Comment by GCuser99** [comment]
fixed in SelenumVBA v6.2 (tB 623).
